src/containers/ProfileSettingsPage/ProfileSettingsForm/ProfileSettingsForm.js

```diff
@@ -19,9 +20,11 @@ import {
   IconSpinner,
   FieldTextInput,
   H4,
+  CustomExtendedDataField,
 } from '../../../components';
 
 import css from './ProfileSettingsForm.module.css';
+import { getPropsForCustomUserFieldInputs } from '../../../util/userHelpers';
```

Comment:
@SariSaar move this to among other **util** imports (to avoid circular references).

Reply:
Note: since userHelpers import types, this import should be after that import statement:
`import { propTypes } from '../../../util/types';`